client/commands/v2/persistent.py

```diff
@@ -503,6 +504,36 @@ def type_errors_to_diagnostics(
     return result
 
 
+def invalid_model_to_diagnostic(
```

Comment:
These two functions are specific to Pysa, so they probably belong in client/commands/v2/pysa_server.py, rather than here